### DISCUSSION TITLE
How about the new logo?

### DISCUSSION THREAD

**Discussion by tomsun28** [apache/hertzbeat#1711] [Ideas]
The design of the logo is a capital letter H with the letter Z in the middle, which represents Hertz frequency and beat.  Thanks @shimingxy suggest. 

<img width="280" alt="Group 6" src="https://github.com/dromara/hertzbeat/assets/24788200/7b1b8ba2-7e59-4755-a7e7-3a550c8b1bf6">

<img width="234" alt="Group 9 (1)" src="https://github.com/dromara/hertzbeat/assets/24788200/b726d2f9-367e-4dd3-bfa9-f0e0a30fbf4a">

how about the logo design? the color?

**Comment by zqr10159** [comment]
I feel like the colors don't match.

**Reply by tomsun28** [reply]
Do we need to unify the colors of these two?

**Reply by zqr10159** [reply]
> Do we need to unify the colors of these two?

Yes, it's better to use soft colors.

**Comment by cuipiheqiuqiu** [comment]
The new logo looks good, but it feels a bit melted rather than beat. I'm not sure if we can make any further adjustments. Also, it's a bit like a mouth or an old-fashioned razor blade. cool

**Comment by zqr10159** [comment]
I agree with you.发自我的 iPhone在 2024年4月16日，16:25，cuipiheqiuqiu ***@***.***> 写道：﻿
The new logo looks good, but it feels a bit melted rather than beat. I'm not sure if we can make any further adjustments. Also, it's a bit like a mouth or an old-fashioned razor blade. cool

**Comment by tomsun28** [comment]
<img width="274" alt="image" src="https://github.com/apache/hertzbeat/assets/24788200/9b859587-c9fa-43ae-a17d-6a0badfc7177">

how about this, The middle is more pointed. The color will choose later.

**Comment by zqr10159** [comment]
If you want the logo to look more throbbing, you can modify the center a bit more, like an EKG在 2024年4月16日，18:17，tomsun28 ***@***.***> 写道：﻿
image.png (view on web)
how about this, The middle is more pointed. The color will choose later.

**Reply by tomsun28** [reply]
Got it.

**Comment by tomsun28** [comment]
<img width="216" alt="image" src="https://github.com/apache/hertzbeat/assets/24788200/d313905a-2da6-4e66-be6b-5c1f22896857">

The new logo looks like an electrocardiogram, but it doesn’t look as good as before. 😂 

I suggest that I sort out a few colors and styles and start voting for everyone to choose. Logo design is too subjective. Let’s change the old logo first and update it later when there is a better design.
